developer-extension/src/panel/components/panel.tsx

```diff
@@ -26,11 +27,17 @@ export function Panel() {
 
   const { events, filters, setFilters, clear } = useEvents(settings)
 
+  const [activeTab, setActiveTab] = useState<string | null>(PanelTabs.Events)
+  useEffect(() => {
+    activeTab && datadogRum.startView(activeTab)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏doing a `startView` in `useEffect` might be too late, as the view is already rendered at this time.

It could be better to:

* start the first view immediately after `RUM.init`, whith the default view name
* start next views in `onTabChange`